drapps/README.md

```diff
@@ -0,0 +1,102 @@
+# DRApps
+
+## About 
+
+DRApps is simple CLI tool for creating your custom application in Data Robot cloud.
+
+Custom application it small web server hosted by Data Robot in Kubernetes environment.
+Data Robot do not provide any storage resources for this server, but you can use Data
+Robot API and other Internet services.
+
+## Installation
+
+For using this CLI script you need only install python requirements with:
+
+    pip install -r requirements.txt
```

Comment:
So to install this, the user need to clone the repository and then go to the cloned destination and install requirements? I think that's too much work and there's a simpler way for the user:

Can we make sure dependencies and the CLI is automatically installed when the user does

```
pip install datarobot-drum
```

This could put a `drapps` script i the user's `$PATH`.

Then the user could just use `drapps` from anywhere, right?

Reply:
In that case I need to make this script part of `datarobot_drum` package.
I though we keep this script as separate entity. At least for now. Am I wrong?

Reply:
If it's hard to make that work, let's keep it as is right now.

But then you should also include a full example of cloning the code, creating a virtualenv and installing dependencies. The users are not going to be Python experts. They're data scientists. They will likely not know how to clone a github repository or install a virtualenv.

They're used to only working in notebooks